### PR TITLE
add License header according to standard template

### PR DESCRIPTION
**Pull request by hairongchen**
Add Apache-2.0 license info

Signed-off-by: Hairong Chen hairong.chen@intel.com

### CODE REVIEW

File: service/quote-server/Makefile [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
```
Comment (Ruoyu-y) [inline service/quote-server/Makefile:1]: remove the first empty line?

**Review by Ruoyu-y — CHANGES_REQUESTED** [review]

**Review by hwang37 — APPROVED** [review]
LGTM

**Review by Ruoyu-y — APPROVED** [review]
LGTM

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**